src/pages/api/github-load-notes.ts

```diff
@@ -88,3 +86,11 @@ ${note.text}\n\n`
     return vtt
   })
 }
+
+export async function loadNotesFromUrl(url: string) {
+  const result = await fetch(url)
+
+  const text = await result.text()
+
+  return parseMdxNotesFile(text)
```

Comment:
now we are just calling the new method and return the value. This doesn't functionally change our code, just means we can test without worrying about loading a note from a URL.

we KNOW `fetch` works and testing that doesn't provide value to our local system.

there are reasons to test this sort of thing, but for now, we can focus on our implementation and not be concerned with testing `fetch` integration.

we'd say `parseMdxNotesFile` has been _isolated for testing_.